tsconfig.build.json

```diff
@@ -12,6 +12,7 @@
 		"removeComments": true,
 		"sourceMap": true,
 		"strictNullChecks": true,
+		"strictBindCallApply": true,
```

Comment:
This means the `.bind(redis)` returns an actual type instead of just `any` which is what you get with this option off